File: Rmm.WebClient/src/app/devicePanel-module/models/deviceSystemLogs.ts
import { DeviceStatus } from '../enums/deviceStatus';
import { DeviceSystemLogsResponse } from './response/deviceSystemLogs-response';

export class DeviceSystemLogs {
    public sourceDeviceId!: string;
    public cpuPerformanceGraph? : number[];
    public terminalLog? : string[];

    constructor(resp: DeviceSystemLogsResponse) {
        this.sourceDeviceId = resp.sourceDeviceId;
        this.cpuPerformanceGraph = resp.cpuPerformanceGraph;
        this.terminalLog = resp.terminalLog;
    }
}

File: Rmm.Domain/Rmm.Domain.Core/Models/DeviceSystemLogs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Rmm.Domain.DAL.Interfaces;
using System.Drawing;
using Rmm.Domain.DAL.Enums;

namespace Rmm.Domain.Core.Models
{
    public class DeviceSystemLogs : IDevicePartialData
    {
        public Guid SourceDeviceId { get; set; }

        public Queue<double>? CpuPerformanceGraph { get; set; }
        public List<string>? TerminalLog { get; set; }
    }
}
